e2e/page-objects/register.po.js

```diff
@@ -0,0 +1,8 @@
+import { Selector } from 'testcafe';
+
+export const RegisterElements = {
+    name: Selector('#name'),
+    email: Selector('#email'),
+    password: Selector('#password'),
+    Btnregister: Selector('button').withText('Register')
```

Comment:
properties in a object should be [camelCase](https://en.wikipedia.org/wiki/Camel_case)
plz replace `Btnregister ` with `btnRegister `